src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -559,9 +559,14 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       "sparse" -> (if (i % 2 == 0) AttributeNull else AttributeNumber(i.toDouble)),
       "vocab1" -> AttributeString(vocab1Strings(i % vocab1Strings.size)),
       "vocab2" -> AttributeString(vocab2Strings(i % vocab2Strings.size)),
-      "mixed" -> (i % 2 match {
+      "mixed" -> (i % 3 match {
```

Comment:
in this case, the lists will be sorted among the other attribute types, with 4 < 420 < 420 900 < 41 < "four"